pinot-common/src/main/java/org/apache/pinot/common/messages/RunPeriodicTaskMessage.java

```diff
@@ -32,20 +32,23 @@ public class RunPeriodicTaskMessage extends Message {
   private static final String PERIODIC_TASK_REQUEST_ID = "requestId";
   private static final String PERIODIC_TASK_NAME_KEY = "taskName";
   private static final String TABLE_NAME_WITH_TYPE_KEY = "tableNameWithType";
+  private static final String TASK_PARAMS = "taskParams";
```

Comment:
Suggest naming it "properties" to be consistent with the name in `RunPeriodicTaskMessageHandler`

Reply:
Ack. Renamed to 'taskProperties'